src/CssInliner.php

```diff
@@ -301,6 +306,36 @@ public function removeExcludedSelector(string $selector): self
         return $this;
     }
 
+    /**
+     * Adds a selector to exclude CSS selector from emogrification.
+     *
+     * @param string $selector the selector to exclude, e.g., ".editor"
+     *
+     * @return $this
+     */
+    public function addExcludedCssSelector(string $selector): self
+    {
+        $this->excludedCssSelectors[$selector] = true;
+
+        return $this;
+    }
+
+    /**
+     * No longer excludes the CSS selector from emogrification.
+     *
+     * @param string $selector the selector to no longer exclude, e.g., ".editor"
+     *
+     * @return $this
+     */
+    public function removeExcludedCssSelector(string $selector): self
```

Comment:
Ditto.

Reply:
Ditto what?

Reply:
Oh, it seems I had clicked on the wrong line, and now I cannot remember what I was referring to in the first place. ;-)